websites/S/Spyke/presence.ts

```diff
@@ -0,0 +1,96 @@
+const presence = new Presence({
+		clientId: "1161544315105976342",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+async function postGQLAPI(
+	operationName: string,
+	query: string,
+	vars: Record<string, unknown>
+) {
+	const response = await fetch("https://api.spyke.social/graphql", {
+		method: "POST",
+		headers: {
+			authorization: `Bearer ${JSON.parse(
+				JSON.parse(localStorage.getItem("persist:Token_Data_Persist"))
+					.jwtIdToken
+			)}`,
+			"Content-Type": "application/json",
+		},
+		body: JSON.stringify({
+			query,
+			variables: vars,
+			operationName,
+		}),
+	});
+	return response.json();
+}
+
+let url = "https://spyke.social/";
+
+presence.on("UpdateData", () => {
+	const nurl = document.URL;
+	if (nurl !== url) {
+		const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/v10pkZA.png",
+			smallImageKey: "https://i.imgur.com/55dCzB1.png",
+			details: "A Community Discussion Platform",
+			state: "On Homepage",
+			startTimestamp: browsingTimestamp,
+		};
+		presenceData.buttons = [
+			{ label: "View Spyke", url: "https://spyke.social/" },
+		];
```

Comment:
[Buttons](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets) that lead of main pages are not allowed.
```suggestion
```